lhotse/kaldi.py

```diff
@@ -7,6 +7,19 @@
 from lhotse.audio import AudioSource, Recording, RecordingSet
 from lhotse.supervision import SupervisionSegment, SupervisionSet
 from lhotse.utils import Pathlike, is_module_available
+from lhotse.audio import _audioread_info 
+
+def from_file(
+    path: Pathlike,
+):  
```

Comment:
Would you mind adding documentation and return type hint to this function?

Reply:
ok,  I will add it.